pytest-selenium/tests/test_attribution.py

```diff
@@ -76,3 +78,58 @@ def test_attribution_collapses_on_navigating_to_new_page(selenium, base_url, boo
 
     # THEN: The citation/attribution section is not open on the new page
     assert not attribution.is_open
+
+    attribution.click_attribution_link()
+
+    # WHEN: Navigating via TOC link
+    if content.is_mobile:
+        toolbar.click_toc_toggle_button()
+
+    toc.sections[-1].click()
+
+    # THEN: The citation/attribution section is not open on the new page
+    assert not attribution.is_open
+
+
+@markers.test_case("C480905")
+@markers.parametrize("page_slug", ["preface"])
+@markers.nondestructive
+def test_book_url_in_citation_text_shows_url_for_default_page(
+    selenium, base_url, book_slug, page_slug
+):
+    # GIVEN: A page is loaded
+    content = Content(selenium, base_url, book_slug=book_slug, page_slug=page_slug).open()
+    attribution = content.attribution
+
+    # WHEN: The attribution section is expanded
+    attribution.click_attribution_link()
+
+    # THEN: The book url in the the citation section should reference the default page of the book
+    default_page_slug = content.default_page(book_slug)
+    attribution_book_url_expected = (
+        "https://openstax.org/books/" + book_slug + "/pages/" + default_page_slug
+    )
+
+    assert attribution_book_url_expected == attribution.book_url
+
+
+@markers.test_case("C480906")
+@markers.parametrize("page_slug", ["preface"])
+@markers.nondestructive
+def test_access_free_url_in_citation_text_shows_url_for_default_page(
+    selenium, base_url, book_slug, page_slug
+):
+    # GIVEN: A page is loaded
+    content = Content(selenium, base_url, book_slug=book_slug, page_slug=page_slug).open()
+    attribution = content.attribution
+
+    # WHEN: The attribution section is expanded
+    attribution.click_attribution_link()
+
+    # THEN: The access for free at url in the the citation section should reference the default page of the book
+    default_page_slug = content.default_page(book_slug)
```

Comment:
the `default_page` utility seems like an unnecessary wrapper, you could make it so you just do `default_page_slug = utils.get_default_page_slug(book_slug)` right here

Reply:
removed the default_page from base/pages .